frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx

```diff
@@ -71,13 +71,28 @@ const filterSelected = (list: BackingStoreStateType[], tableId: number) => {
   return list.filter((e) => e.selectedBy === tableId || e.selectedBy === '').sort(sort);
 };
 
-const getTableRows = (list: K8sResourceKind[]) => {
+const findInTier = (tier: string[], name: string) => tier.find((elem) => elem === name);
```

Comment:
Add return type i.e. `string[]`

Reply:
It's implicit. FYI the return type is `string`